src/test/java/no/unit/nva/model/instancetypes/ReportPolicyTest.java

```diff
@@ -0,0 +1,63 @@
+package no.unit.nva.model.instancetypes;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Range;
+import no.unit.nva.model.util.ReportContentTestUtil;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ReportPolicyTest {
+
+    public static final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @DisplayName("Report policy can be created from JSON")
+    @Test
+    void reportPolicyReturnsObjectWhenJsonInputIsCorrectlySerialized() throws JsonProcessingException,
+            InvalidPageTypeException {
+        ReportPolicy expected = generateReportPolicy("42", "1", "3", false);
+        String json = objectMapper.writeValueAsString(expected);
+        ReportPolicy reportPolicy = objectMapper.readValue(json, ReportPolicy.class);
+        assertEquals(expected, reportPolicy);
+    }
+
+    @DisplayName("Report policy cannot be peer reviewed")
+    @Test
+    void reportPolicySetsPeerReviewedToFalseWhenPeerReviewIsTrue() throws JsonProcessingException,
+            InvalidPageTypeException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        String type = "ReportPolicy";
+        String pages = "42";
+        String introductionBegin = "1";
+        String introductionEnd = "3";
+        boolean illustrated = false;
+        ReportPolicy expected = generateReportPolicy(pages, introductionBegin, introductionEnd, illustrated);
+
+        String json = ReportContentTestUtil.generateJsonString(type, pages, introductionBegin, introductionEnd,
+                illustrated, false);
+        assertEquals(expected, objectMapper.readValue(json, ReportPolicy.class));
```

Comment:
Two exceptions can occur here. One while trying to read an invalid ReportPolicy.json file and another if the condition fails. Assign the actual value to a variable.